### ISSUE TITLE
Create conan package for cpp-mapreduce-client

### ISSUE THREAD

**Issue by desertfury** [ytsaurus/ytsaurus#421]
ytsaurus itself uses conan package manager for building. But there is no conan recipe for cpp-mapreduce-client. So for now if one wants to use cpp sdk they need to copy/paste ytsaurus sources or add ytsaurus repo as submodule.

It would be nice if we could use cpp-mapreduce-client library as conan package.

**Comment by Kontakter** [comment]
Yes we want to do that, but the first thing that we are working on – it is creating a separate repository with SDK and without bundled protobuf and libstd++
